scripts/test_check_backend_patch.py

```
@@ -235,23 +235,64 @@ def _ignore(submodule: Path, pattern: str) -> None:
         capture_output=True,
         text=True,
+        env=environment,
     )
 
 
+def _shipped_submodule_fixture(directory: str):
+    """Clone the real pinned backend locally so the production entry can be mutated without touching the checkout."""
+    source = ROOT / MODULE.SUBMODULE
+    assert (source / ".git").exists(), "the SafetyHook submodule must be initialized"
+    checkout = Path(directory) / "safetyhook"
+    subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)